docs/source/plugins/using_plugins.rst

```diff
@@ -0,0 +1,746 @@
+        fiftyone plugins download \
+            https://github.com/<user>/<repo>[/tree/branch] \
+            --plugin-names <name1> <name2> <name3>
+
+  .. group-tab:: Python
+
+    .. code-block:: python
+        :linenos:
+
+        import fiftyone.plugins as fop
+
+        # Download plugin(s) from a GitHub repository
+        fop.download_plugin("https://github.com/<user>/<repo>[/tree/branch]")
+
+        # Download plugin(b) by specifying the GitHub repository details
+        fop.download_plugin("<user>/<repo>[/<ref>]")
+
+        # Download specific plugins from a GitHub repository
+        fop.download_plugin(url_or_gh_repo, plugin_names=["<name1>", "<name2>"])
+
+.. note::
+
+    A GitHub repository may contain multiple plugins. By default, all plugins
+    that are found within the first three directory levels are installed, but
+    you can select specific ones if desired as shown above.
+
+.. note::
+
+    All plugins are downloaded to your
+    :ref:`plugins directory <plugins-directory>`.
+
+.. note::
+
+    You can download plugins from private GitHub repositories that you have
+    access to by providing your GitHub personal access token via the
+    `GITHUB_TOKEN` environment variable.
+
+.. _plugins-directory:
+
+Your plugins directory
+______________________
+
+All plugins must be stored (and are automatically
+:ref:`downloaded to <plugins-download>` your plugins directory in order for
+FiftyOne to find them.
+
+By default, plugins are downloaded to `~/fiftyone/__plugins__`, but you can
+customize this directory by setting the `FIFTYONE_PLUGINS_DIR` environment
+variable:
+
+.. code-block:: shell
+
+    export FIFTYONE_PLUGINS_DIR=/path/to/your/plugins
+
+You can also permanently configure this directory by adding it to your
+:ref:`FiftyOne config <configuring-fiftyone>`.
+
+.. code-block:: json
+
+    {
+        "plugins_dir": "/path/to/your/plugins"
+    }
+
+You can locate your current plugins directory by running the following command:
+
+.. code-block:: shell
+
+    fiftyone config plugins_dir
+    # ~/fiftyone/__plugins__
+
+.. note::
+
+    Your plugins directory must be readable by the FiftyOne server.
+
+.. _plugins-managing:
+
+Managing plugins
+________________
+
+You can use the :ref:`fiftyone plugins <cli-fiftyone-plugins>` and
+:ref:`fiftyone operators <cli-fiftyone-operators>` CLI methods to perform a
+variety of plugin-related actions.
+
+Listing plugins
+---------------
+
+You can use the :ref:`fiftyone plugins list <cli-fiftyone-plugins-list>`
+command to list the plugins that you've downloaded or created locally:
+
+.. code-block:: shell
+
+    # List all locally available plugins
+    fiftyone plugins list
+
+.. code-block:: shell
+
+    # List enabled plugins
+    fiftyone plugins list --enabled
+
+.. code-block:: shell
+
+    # List disabled plugins
+    fiftyone plugins list --disabled
+
+.. code-block:: text
+
+    plugin               version   enabled  directory
```

Comment:
FWIW this doesn't seem to match the changed version proposed in this PR. Unless I'm missing something I thought 
it would have an "on startup" column.